packages/cubejs-schema-compiler/adapter/BaseQuery.js

```diff
@@ -113,7 +113,7 @@ class BaseQuery {
         throw new UserError(`You cannot use dimension and measure in same condition: ${JSON.stringify(f)}`);
       }
 
-      if (!f.dimension && !f.member) {
+      if (!f.member && !f.dimension) {
```

Comment:
A & B == B & A (associative operation), Why did you change it?)

Reply:
Similar case https://github.com/cube-js/cube.js/blob/master/packages/cubejs-schema-compiler/adapter/BaseQuery.js#L63

Reply:
@ovr the idea behind the change is this warning https://github.com/cube-js/cube.js/blob/054c3357ad2f1ad7d6ffb2312c43299e738774d1/packages/cubejs-api-gateway/index.js#L305

If we check for `.member` first and it's there we skip `.dimension` call and therefore avoid printing the warning to the console